src/RefreshManager.m

```diff
@@ -33,6 +33,7 @@
 #import "NSNotificationAdditions.h"
 #import "VTPG_Common.h"
 #import "FeedItem.h"
+#import <QuartzCore/QuartzCore.h>
```

Comment:
My [issue-833bis branch](https://github.com/barijaona/vienna-rss/tree/issue-833bis) might be slightly more elegant than this one. I'll close this one and submit a new pull request.